ngu/lib_secp256k1.c

```diff
@@ -1,12 +1,12 @@
 #ifndef NO_QSTR
 
-# ifndef HAVE_CONFIG_H
```

Comment:
configuration header was removed in `v0.3.0` https://github.com/bitcoin-core/secp256k1/blob/master/CHANGELOG.md#removed